app/plugin.go

```diff
@@ -103,6 +104,7 @@ func (a *App) SyncPluginsActiveState() {
 					message.Add("manifest", updatedManifest.ClientManifest())
 					a.Publish(message)
 				}
+				pluginsEnvironment.SetPluginState(pluginId, model.PluginStateRunning)
 			}
```

Comment:
Please hold off on merging. I'm thinking this line may need to be removed, as it is flagging the plugin as active outside of `env.Activate`. I haven't run into any issues in testing, but I want to trace the code paths again.